.cargo/config.toml

```diff
@@ -1,2 +1,26 @@
 [alias]
 xtask = "run --package xtask --"
+
+# On Windows 
+# ```
+# cargo install -f cargo-binutils
+# rustup component add llvm-tools-preview
+# ```
+[target.x86_64-pc-windows-msvc]
+rustflags = ["-C", "link-arg=-fuse-ld=lld"]
+
+[target.x86_64-pc-windows-gnu]
+rustflags = ["-C", "link-arg=-fuse-ld=lld"]
+
+# On Linux:
+# - Ubuntu, `sudo apt-get install lld clang`
+# - Arch, `sudo pacman -S lld clang`
+[target.x86_64-unknown-linux-gnu]
```

Comment:
Can you also add:

* `target.aarch64-unknown-linux-gnu`
* `target.x86_64-unknown-linux-musl`
* `target.aarch64-unknown-linux-musl`

With the same parameters? [`musl`](http://musl.libc.org/) is the alternative C library used mostly in Alpine Linux.